tests/command_count_test.go

```diff
@@ -0,0 +1,41 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestCommandCOUNT(t *testing.T) {
+	commandsCount := 40
+
+	subscriber := getLocalConnection()
+
+	responseValue := fireCommand(subscriber, "COUNT")
+	responseCount  := responseValue.(int64)
+	if responseValue == nil {
+		t.Fail()
+	}
+
+	assert.Assert(t, responseCount == int64(commandsCount),
```

Comment:
instead of typecasting this to `int64` you can set commandCount as `int64` while declaring. Also, is this a good idea to test this? So, every time you add a new command, this test is bound to fail. Can you confirm how is Redis codebase testing this?

Reply:
May i know where can I find the unit tests in redis codebase?